app/jobs/trainees/create_from_dttp_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Trainees
+  class CreateFromDttpJob < ApplicationJob
+    queue_as :default
+
+    def perform
+      return unless FeatureService.enabled?("import_trainees_from_dttp")
+
+      Dttp::Trainee.joins(:provider).includes(:placement_assignments).unprocessed.each do |dttp_trainee|
+        CreateFromDttp.call(dttp_trainee: dttp_trainee)
+      rescue Trainees::CreateFromDttp::UnrecognisedStatusError => e
+        Sentry.capture_exception(e)
```

Comment:
Should we update the state on the record here?

Reply:
Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/1787/commits/e419d8374f715a3788a245ea5cccda0efaa07a7d